pkg/lightning/backend/local/duplicate.go

```diff
@@ -1074,6 +1074,37 @@ type DupeController struct {
 	importClientFactory importClientFactory
 	resourceGroupName   string
 	taskType            string
+
+	collectRemoteDupRows bool
+}
+
+// NewDupeController creates a new DupeController.
+func NewDupeController(
```

Comment:
```suggestion
func NewDupeControllerForRemoteBackend(
```
please also simplify the params, this is a function specific for remote backend